pkg/yurthub/filter/filter.go

```diff
@@ -238,6 +238,11 @@ func (frc *filterReadCloser) StreamResponseFilter(rc io.ReadCloser, ch chan *byt
 			return err
 		}
 
+		// BOOKMARK and ERROR response are unnecessary to filter
+		if watchType == watch.Bookmark || watchType == watch.Error {
+			continue
+		}
+
 		newObj := frc.objectFilter.Filter(obj, frc.stopCh)
 		if yurtutil.IsNil(newObj) {
 			continue
```

Comment:
we should make sure that `BOOKMARK` and `ERROR` response can be passed through to the clients(like kube-proxy), so we should improve the code as following:

```
newObj := obj
if watchType != watch.Bookmark && watchType != watch.Error {
	newObj = frc.objectFilter.Filter(obj, frc.stopCh)
	if yurtutil.IsNil(newObj) {
		continue
        }
}
```